AuxTel/AuxTel-Standard-Operations/AuxTel-Daytime-Operations/Daytime-Checkout.rst

```diff
@@ -22,58 +28,58 @@ but offer some flexibility in what state to leave the Auxiliary Telescope
 They can also be run individually if a single component is undergoing maintenance 
 and requires a check of basic functionality. 
 
-None of the scripts require configuration. 
+**None of the scripts require configuration.**
 
 .. Important::
 
     The daytime checkout scripts involve Telescope, Dome, and Instrument component movement. 
     Before running, ensure that the area surrounding the telescope is and dome is clear for movement. 
     You MUST announce that you are running the checks on the #summit-announce slack channel before execution. 
 
-The scripts can all be found under the Standard scripts panel in the LOVE ATQueue view. 
+The scripts can all be found under the standard scripts panel in the *LOVE ATQueue* view. 
 The order of script execution is nominally:
 
-* auxtel/enable_latiss.py - Can be skipped if LATISS enabled 
-* auxtel/daytime_checkout/latiss_checkout.py
-* auxtel/enable_atcs - Can be skipped if ATCS already enabled
-* auxtel/daytime_checkout/atpneumatics_checkout.py
-* auxtel/daytime_checkout/telescope_and_dome_checkout.py
-* auxtel/daytime_checkout/slew_and_take_image_checkout.py
-* auxtel/prepare_for/flats - Can be substituted for other prepare_for scripts to leave AuxTel in different state
+* :file:`auxtel/enable_latiss.py` - Can be skipped if LATISS enabled 
+* :file:`auxtel/daytime_checkout/latiss_checkout.py`
+* :file:`auxtel/enable_atcs.py` - Can be skipped if ATCS already enabled
+* :file:`auxtel/daytime_checkout/atpneumatics_checkout.py`
+* :file:`auxtel/daytime_checkout/telescope_and_dome_checkout.py`
+* :file:`auxtel/daytime_checkout/slew_and_take_image_checkout.py`
 
 *auxtel/enable_latiss.py*
 =========================
 
-Bring LATISS to an enabled state to prepare for checkout. 
+Bring *LATISS* to an enabled state to prepare for checkout. 
 
 *auxtel/daytime_checkout/latiss_checkout.py*
 ============================================
 
-This script will take a bias and engineering frame with LATISS and ensure that they are ingested by the OODS.
+This script will take a bias and an engineering frame with *LATISS* 
+and ensure that they are ingested by the *OODS*.
 
-Check that the bias pops up in the monitor (https://roundtable.lsst.codes/rubintv/summit/auxtel/monitor_current)
-If it fails, let Patrick or Merlin know.
+Verify that the bias appears on the `summit's RubinTV`_. 
+In case the image fails to appear, promptly communicate the issue in the Slack channel *#summit-auxtel*. 
```

Comment:
Until now all the channels are called as channel X. Mentioning Slack tie the project to a certain vendor.

Reply:
Good call. Thanks! I have removed two mentions to the vendor.